parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -18,6 +18,10 @@ ExecStartPre=/bin/mount --make-shared /var/lib/kubelet
 ExecStartPre=-/sbin/ebtables -t nat --list
 ExecStartPre=-/sbin/iptables -t nat --numeric --list
 
+{{- if and NeedsContainerd IsKubenet }}{{- if not HasCalicoNetworkPolicy}}
```

Comment:
just curious - does calico network provider not using promsc mode or does it already have dedup rule?